### DISCUSSION TITLE
Customize Microsoft Teams contact point header message

### DISCUSSION THREAD

**Discussion by AlbMor** [grafana/grafana#40475] [Alerting]
## What Grafana version and what operating system are you using?
Docker: grafana/grafana:8.1.2 on Windows Docker Desktop with wsl2

## What are you trying to achieve?
Send Alert to Microsoft Team contact point and have a desired Header message

## How are you trying to achieve it?
Using the new Grafana Managed Alert system

## What happened?
In the Microsoft Team contact point the header message is created using the alert label.

## What did you expect to happen?
I would like to have control over the header message of the Microsoft Teams contact point. For example the Slack contact point has Title parameter that can be set.

**Comment by ristoporila** [comment]
I have already made the same feature request: https://github.com/grafana/grafana/discussions/38526

**Reply by AlbMor** [reply]
@ristoporila nice

**Comment by mucho-al-dente** [comment]
This has been addressed in e8f4b58a8ba40812c938942442ca8fb552a9360f and merged into `main`

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
